LICENSE.txt

```diff
@@ -0,0 +1,232 @@
+Copyright © 2014-2021 Cask Data, Inc.
```

Comment:
Should this still be Cask Data?

Reply:
All current licenses are that way, hence kept it the same.